doc/Makefile

```diff
@@ -237,7 +279,7 @@ hpp-statistics.configure.dep: hpp-util.install hpp-statistics.checkout
 hpp-core.configure.dep: hpp-constraints.install hpp-statistics.install \
 	hpp-core.checkout
 hpp-constraints.configure.dep: hpp-pinocchio.install hpp-statistics.install \
-	hpp-constraints.checkout
+	romeo.install hpp-constraints.checkout
```

Comment:
I don't really like to have this dependency. I know this is not your fault and that romeo.install is required for the unit-tests but I don't think the installation of HPP should trigger the installation of romeo.

After checking the unit-tests, they could work just fine with HumanoidSimple rather than HumanoidRomeo. I will update them soon.

I hesitate between taking this as @pFernbach proposes and keeping it as it is, while remove the dependency in the CMake of hpp-constraints.

@florent-lamiraux @nim65s I think you discussed about this issue but can't remember the conclusions.

Reply:
See #https://github.com/humanoid-path-planner/hpp-constraints/issues/64

Reply:
I think that:
- for short term, we can use robotpkg-romeo-description
- for long term, we should work at removing all robot descriptions other than simple humanoid, simple arm & simple quadruped (the last ones still have to be created) from all unit tests, but we can still use many real robot descriptions for benchmarks

Reply:
I pushed changes to use  robotpkg-romeo-description

Reply:
Ok, I will re-generate the docker images then

Reply:
Thanks.

Reply:
The images for devel are ready. @pFernbach : can you push your devel branch to your gepgitlab repo, to check everything ?

Reply:
https://gepgitlab.laas.fr/pfernbac/hpp-doc/pipelines/2775